hotness/buildsys.py

```diff
@@ -29,6 +47,12 @@ def __init__(self, consumer, config):
         self.ca_cert = config['ca_cert']
         self.git_url = config['git_url']
         self.userstring = config['userstring']
+        if isinstance(self.userstring, six.string_types):
+            msg = ('Using a string for "userstring" in "hotness.koji"'
+                   ' is deprecated, please use a tuple')
+            warnings.warn(msg, DeprecationWarning)
+        else:
+            self.userstring = ' '.join(self.userstring)
```

Comment:
Are we here converting back the tuple to string?

If that's the case, wouldn't it be more logical to deprecate the support for tuple rather than the one for string?